docs/algorithms/sig/sphincs.yml

```diff
@@ -141,7 +142,7 @@ parameter-sets:
     no-secret-dependent-branching-checked-by-valgrind: true
     large-stack-usage: false
 - name: SPHINCS+-Haraka-192f-robust
-  claimed-nist-level: 3
+  claimed-nist-level: 2
```

Comment:
@dstebila Why is there still a reference to a Haraka version? And why did the claimed NIST level change? Would it be more appropriate to remove all references to all no-longer supported SPHINCS+ variants from this YML file? FWIW, I did the same for Kyber90s and DilithiumAES as `copy_from_upstream` did not have the automation for this.

Reply:
I didn't realize that copy_from_upstream wouldn't clean that all up.  I'll make a TODO to search for references to the removed algorithms.

Reply:
> I didn't realize that copy_from_upstream wouldn't clean that all up. 

I didn't either when doing https://github.com/open-quantum-safe/liboqs/pull/1455. All it needed was removal of the no longer needed algs from https://github.com/open-quantum-safe/liboqs/blob/main/docs/algorithms/sig/sphincs.yml (in this case) and re-running `copy_from_upstream`.